### DISCUSSION TITLE
CPX: ESP32: W (2504/2514) ROUTER: Cannot route from GAP8 [0x00] to [0x00]

### DISCUSSION THREAD

**Discussion by SUJUSTDO** [bitcraze/discussions#1618] [Q&A] [answered]
When I burned a custom model to the AI-deck via crazyradio, the following error appeared on the Console. What is the reason for this?
CPX: GAP8: Starting to open camera
CPX: GAP8: Opened Camera
CPX: GAP8: Allocated memory for camera buffer
CPX: GAP8: Allocated memory for output
CPX: GAP8: Allocated memory for task
CPX: GAP8: Constructed CNN
CPX: ESP32: W (2504) ROUTER: Cannot route from GAP8 [0x00] to [0x00]
CPX: ESP32: W (2514) ROUTER: Cannot route from GAP8 [0x00] to [0x00]
STAB: Starting stabilizer loop
SUP: Can not fly
SUP: Ready to fly

**Comment by gemenerik** [comment, marked as answer]
It seems like CPX might not be working correctly on the GAP8 side. You may be missing the CPX implementation in your GAP8 application. For reference, take a look at the [classification example,](https://github.com/bitcraze/aideck-gap8-examples/blob/master/examples/ai/classification/classification.c) which includes the CPX initialization you are probably missing.

**Comment by SUJUSTDO** [comment]
Thank you for your reply.
